plugins/dynatrace/src/components/DynatraceTab/DynatraceTab.tsx

```diff
@@ -18,37 +18,57 @@ import { Grid } from '@material-ui/core';
 import {
   Page,
   Content,
-  ContentHeader,
-  SupportButton,
   MissingAnnotationEmptyState,
 } from '@backstage/core-components';
 import { useEntity } from '@backstage/plugin-catalog-react';
+import { useApi, configApiRef } from '@backstage/core-plugin-api';
 import { ProblemsList } from '../Problems/ProblemsList';
+import { SyntheticsCard } from '../Synthetics/SyntheticsCard';
 import { isDynatraceAvailable } from '../../plugin';
-import { DYNATRACE_ID_ANNOTATION } from '../../constants';
+import {
+  DYNATRACE_ID_ANNOTATION,
+  DYNATRACE_SYNTHETICS_ANNOTATION,
+} from '../../constants';
 
 export const DynatraceTab = () => {
   const { entity } = useEntity();
 
+  const configApi = useApi(configApiRef);
+  const dynatraceBaseUrl = configApi.getString('dynatrace.baseUrl');
+
   if (!isDynatraceAvailable(entity)) {
     return <MissingAnnotationEmptyState annotation={DYNATRACE_ID_ANNOTATION} />;
   }
 
   const dynatraceEntityId: string =
     entity?.metadata.annotations?.[DYNATRACE_ID_ANNOTATION]!;
 
+  const syntheticsIds: string =
+    entity?.metadata.annotations?.[DYNATRACE_SYNTHETICS_ANNOTATION]!;
+
   return (
     <Page themeId="tool">
       <Content>
-        <ContentHeader title="Dynatrace">
-          <SupportButton>
-            Plugin to show information from Dynatrace
-          </SupportButton>
-        </ContentHeader>
         <Grid container spacing={2}>
           <Grid item xs={12} lg={12}>
-            <ProblemsList dynatraceEntityId={dynatraceEntityId} />
+            <ProblemsList
+              dynatraceEntityId={dynatraceEntityId}
+              dynatraceBaseUrl={dynatraceBaseUrl}
+            />
           </Grid>
+          {syntheticsIds
+            .replace(' ', '')
+            .split(',')
+            .map(id => {
+              return (
+                <Grid item xs={12} lg={12}>
+                  <SyntheticsCard
+                    syntheticsId={id}
+                    dynatraceBaseUrl={dynatraceBaseUrl}
```

Comment:
Couldn't this component just grab that itself right out of the config, instead of having to pass it down like this? Would probably be cleaner. You could even make a `useDynatraceBaseUrl` hook if you don't want to repeat it in the two components.